.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,17 @@
-## What does this PR do
-
+# What does this PR do?
 
 ## Standards checklist
 
-- [ ] The PR title is descriptive
+- [ ] The PR title is descriptive and scoped (e.g., area: short summary)
 - [ ] I have read `CONTRIBUTING.md`
-- [ ] *Optional:* I have tested the code myself
-- [ ] If this PR introduces new user-facing messages they are translated
+- [ ] Optional: I have tested the code myself
+- [ ] If this PR introduces user-facing messages, they are translated
 
 ## For new steps
 
-- [ ] *Optional:* Topgrade skips this step where needed
-- [ ] *Optional:* The `--dry-run` option works with this step
-- [ ] *Optional:* The `--yes` option works with this step if it is supported by
-  the underlying command
+- [ ] Optional: Topgrade skips this step where needed
+- [ ] Optional: The `--dry-run` option works with this step
+- [ ] Optional: The `--yes` option works with this step if it is supported by the underlying command
 
-If you developed a feature or a bug fix for someone else and you do not have the
-means to test it, please tag this person here.
+If you developed a feature or a bug fix for someone else and you do not have the means to test it, please tag this
+person here.
```

Comment:
Why is this in this PR? I'm not against these wording changes, but the PR is large enough without them.